VRoidHubLoader/Modules/VrmLoaderModule.cs

```diff
@@ -62,17 +63,37 @@ private void Update()
         {
             Core.Msg("OnUpdate: VrmLoaderModule F4 pressed");
 
-            // MenuManager is a singleton? sweet.
-            if (!MenuManager.Instance.IsOpen)
+            await ImportVRM();
```

Comment:
Also, in the case of importing the model with F4, could we also equip the model automatically to keep the old F4 behaviour after it is added to the VRM folder?

Reply:
Restored functionality in [0a993f6](https://github.com/YusufOzmen01/desktopmate-custom-avatar-loader/pull/82/commits/0a993f61a4360f3c4ba28ecc3c82f711fe4b98cb)